service/history/replication/poller_manager.go

```diff
@@ -64,7 +70,8 @@ func (p pollerManagerImpl) getPollingShardIDs(remoteClusterName string) []int32
 func generatePollingShardIDs(localShardId int32, localShardCount int32, remoteShardCount int32) []int32 {
```

Comment:
get source / target shard IDs? 
polling shard IDs can be mis-read?

Reply:
updated